Modules/Core/Sources/Core/MediaType/MediaType.swift

```diff
@@ -1,7 +1,7 @@
-public protocol MediaTypeRepresentor {
+public protocol MediaTypeConvertible {
```

Comment:
What's the reasoning behind this? `MediaTypeRepresentable` seems to best reflect the meaning here given that we established convertible to mean initializable + representable.